ControlMeasurements/Views/Shared/_Layout.cshtml

```diff
@@ -26,10 +28,9 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">ControlMeasurements</a>
-                <a asp-area="" asp-controller="Heating" asp-action="Index" class="navbar-brand">HeatingMeasurements</a>
+                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">Summary</a>
+                <a asp-area="" asp-controller="Measure" asp-action="Index" class="navbar-brand">ControlMeasurements</a>
```

Comment:
Chyba nie powinienes miec dwoch linkow z klasa navbar-brand, ale to jest do doczytania w dokumentacji Bootstrapa :)

Reply:
a oki to doczytam i pozniej zmienie :)